releasenotes/notes/jmxfetch-kill-6dcb1a0b9ab31ca5.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    Fixed JMXFetch process not being terminated on Windows in certain cases.
```

Comment:
If you plan to merge multiple PRs that will fix the same issue (from the user POV) I think we should probably add only one release note that informs that JMXFetch now properly exits on windows.

Reply:
This is the only PR regarding JMX on Windows.

Reply:
Ok my bad then, feel free to merge